File: Exercicios/Lista01/ex1.cpp
#include <iostream>
#include <iomanip>

using namespace std;
class cerchio {
    public:
    double r, p = 3.14;

    double calc_area(){
        double area = r * r * p;
        return area;
    }

    double calc_circun(){
        double circ = p * 2 * r;
        return circ;
    }
};

int main() {
    cerchio c;
    cout << "Informe o valor do raio: ";
    cin >> c.r;
    double area = c.calc_area();
    double circ = c.calc_circun();

    cout << fixed << setprecision(2);
    cout << "Área: " << area << endl;
    cout << "Circunferência: " << circ <<endl;

    return 0;
}

File: Exercicios/Lista01/Ex1.java
import java.util.Scanner;

class Circulo {
    public double r, p = 3.14;

    public double calc_area() {
        double area = r * r * p;
        return area; 
    }

    public double calc_circ() {
        double circ = p * 2 * r;
        return circ;
    }
}

public class Ex1 {

    public static void main(String[] args) {
        Circulo x = new Circulo();
        System.out.print("Informe o valor do raio: ");
        Scanner num = new Scanner(System.in);
        x.r = num.nextDouble();
        double area = x.calc_area();
        double circ = x.calc_circ();
        System.out.println(String.format("O valor de área é: %.2f", area));
        System.out.println(String.format("O valor da circunferência é: %.2f",circ));
        num.close();
    }
}